astro.config.ts

```diff
@@ -43,6 +43,10 @@ export default defineConfig({
 	legacy: {
 		astroFlavoredMarkdown: true,
 	},
+	server: {
+		// avoid  error   getaddrinfo ENOTFOUND localhost
+		host: true
+	},
```

Comment:
I don’t think we want to set this by default: it will automatically expose the dev server which can be a security risk in some circumstances. If people are running into this, we might want to document using the `--host` CLI flag to opt in to this behaviour.

```suggestion
```